ambry-utils/src/test/java/com.github.ambry.utils/UtilsTest.java

```diff
@@ -130,6 +148,19 @@ public void testReadBuffers()
     for (int i = 0; i < 40000; i++) {
       Assert.assertEquals(buf[i + 4], outputBuf.array()[i]);
     }
+    // 0 size
+    inputBuf.rewind();
+    inputBuf.putInt(0, 0);
```

Comment:
Minor: Do you have to specify the index for `putInt()`? In all these places you seem to be rewinding anyway?